SPECS-EXTENDED/python-cheetah/python-cheetah.spec

```diff
@@ -1,109 +1,144 @@
-%bcond_with python2
-
 Name:           python-cheetah
-Version:        3.2.4
-Release:        4%{?dist}
+Version:        3.2.6.post1
+Release:        12%{?dist}
 Summary:        Template engine and code generator
 
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://cheetahtemplate.org/
-Source:         https://github.com/CheetahTemplate3/cheetah3/archive/%{version}/%{name}-%{version}.tar.gz
+Source:         https://github.com/CheetahTemplate3/cheetah3/archive/%{version}/Cheetah3-%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+# Instead of playing Whac-A-Mole and adding more and more basepythons,
+# e.g. in https://github.com/CheetahTemplate3/cheetah3/commit/6be6bc10a4,
+# we let tox do the right thing by not setting any:
+Patch:          tox-no-basepython.patch
+Patch1:         cheetah3-3.2.6.post1-protect-cgi.patch
+Patch2:         cheetah3-3.2.6.post1-loadTestsFromModule.patch
+Patch3:         cheetah3-3.2.6.post1-typeerror.patch
+Patch4:         cheetah3-3.2.6.post1-framelocalsproxy.patch
+Patch5:         cheetah3-3.2.6.post1-parse_qs.patch
+
 BuildRequires:  gcc
+BuildRequires:  python3-devel
+BuildRequires:  python3-pip
+BuildRequires:  python3-wheel
+
+# for tests
+%if 0%{?with_check}
+BuildRequires:  python3-tox
+BuildRequires:  python3-pluggy
+BuildRequires:  python3-py
+BuildRequires:  python3-filelock
+BuildRequires:  python3-toml
```

Comment:
Please recheck the dependencies which are required for building the package and should be moved out from under with_check macro.

Re-run of BB is failing with these changes: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=749101&view=logs&j=bac295d9-f2e0-5165-e63e-e76f383e1a27